test/hotspot/gtest/oops/test_objArrayOop.cpp

```diff
@@ -28,29 +28,41 @@
   } x[] = {
-//    ObjAligInB, UseCCP, UseCoops, object size in heap words
+//    ObjAligInB, UseCCP, UseCoops, UseCOH, object size in heap words
 #ifdef _LP64
-    { 8,          false,  false,    4 },  // 20 byte header, 8 byte oops
-    { 8,          false,  true,     3 },  // 20 byte header, 4 byte oops
-    { 8,          true,   false,    3 },  // 16 byte header, 8 byte oops
-    { 8,          true,   true,     3 },  // 16 byte header, 4 byte oops
-    { 16,         false,  false,    4 },  // 20 byte header, 8 byte oops, 16-byte align
-    { 16,         false,  true,     4 },  // 20 byte header, 4 byte oops, 16-byte align
-    { 16,         true,   false,    4 },  // 16 byte header, 8 byte oops, 16-byte align
-    { 16,         true,   true,     4 },  // 16 byte header, 4 byte oops, 16-byte align
-    { 256,        false,  false,    32 }, // 20 byte header, 8 byte oops, 256-byte align
-    { 256,        false,  true,     32 }, // 20 byte header, 4 byte oops, 256-byte align
-    { 256,        true,   false,    32 }, // 16 byte header, 8 byte oops, 256-byte align
-    { 256,        true,   true,     32 }, // 16 byte header, 4 byte oops, 256-byte align
+    { 8,          false,  false,    false,  4 },  // 20 byte header, 8 byte oops
+    { 8,          false,  true,     false,  3 },  // 20 byte header, 4 byte oops
+    { 8,          true,   false,    false,  3 },  // 16 byte header, 8 byte oops
+    { 8,          true,   true,     false,  3 },  // 16 byte header, 4 byte oops
+    { 16,         false,  false,    false,  4 },  // 20 byte header, 8 byte oops, 16-byte align
+    { 16,         false,  true,     false,  4 },  // 20 byte header, 4 byte oops, 16-byte align
+    { 16,         true,   false,    false,  4 },  // 16 byte header, 8 byte oops, 16-byte align
+    { 16,         true,   true,     false,  4 },  // 16 byte header, 4 byte oops, 16-byte align
+    { 256,        false,  false,    false,  32 }, // 20 byte header, 8 byte oops, 256-byte align
+    { 256,        false,  true,     false,  32 }, // 20 byte header, 4 byte oops, 256-byte align
+    { 256,        true,   false,    false,  32 }, // 16 byte header, 8 byte oops, 256-byte align
+    { 256,        true,   true,     false,  32 }, // 16 byte header, 4 byte oops, 256-byte align
+    { 8,          false,  false,    true,   3 },  // 16 byte header, 8 byte oops
+    { 8,          false,  true,     true,   2 },  // 12 byte header, 4 byte oops
+    { 8,          true,   false,    true,   3 },  // 16 byte header, 8 byte oops
+    { 8,          true,   true,     true,   2 },  // 12 byte header, 4 byte oops
+    { 16,         false,  false,    true,   4 },  // 20 byte header, 8 byte oops, 16-byte align
+    { 16,         false,  true,     true,   2 },  // 20 byte header, 4 byte oops, 16-byte align
+    { 16,         true,   false,    true,   4 },  // 16 byte header, 8 byte oops, 16-byte align
+    { 16,         true,   true,     true,   2 },  // 16 byte header, 4 byte oops, 16-byte align
```

Comment:
Are these really "20 byte" and "16 byte headers" here? Thought they would be smaller.